student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(id, name, classroom, age = 'unknown', parent_permission: true)
+    @classroom = classroom
+    super(id, name, age, parent_permission)
+  end
```

Comment:
- [ ] Kindly improve this constructor to match the Person parent class.

- [ ] The `parent_permission` parameter uses colon notation so it's assignment is somewhat different

  ```
  def initialize(classroom, age, name = "unknown", parent_permission: true)
    @classroom = classroom
    super(age, name, parent_permission: parent_permission)
  end
  ```